branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -37,7 +37,14 @@ public IBranchLayout slideOut(String branchName) throws EntryDoesNotExistExcepti
     if (entryToSlideOut == null) {
       throw new EntryDoesNotExistException("Branch entry '${branchName}' does not exist");
     }
-    return new BranchLayout(rootEntries.flatMap(rootEntry -> slideOut(rootEntry, entryToSlideOut)));
+
+    var newBranchLayout = new BranchLayout(rootEntries.flatMap(rootEntry -> slideOut(rootEntry, entryToSlideOut)));
```

Comment:
Hmm I don't like this double slide-out here.... how about just sth like:

`return new BranchLayout(rootEntries.flatMap(rootEntry -> slideOut(rootEntry, branchName)));`

as in, branches just slid out based on their matching name, with no need to perform `findEntryByName` in the first place?

Also, if you could a test that would've captured the bug in the previous implementation (i.e. the one where only one of duplicated entries is being slid out)